netlify.toml

```diff
@@ -0,0 +1,46 @@
+########################################
+# Bonita redirects
+########################################
+[[redirects]]
+  from = "/bonita/latest/*"
+# TODO if we want proxy/rewrite instead, add status = 200 (https://docs.netlify.com/routing/redirects/rewrites-proxies/)
+  to = "/bonita/7.12/:splat"
+
+[[redirects]]
+  from = "/bonita/"
+  to = "/bonita/7.12/"
```

Comment:
ℹ️ the trailing slash avoids an extra redirect as netlify already redirects `/bonita/7.12` to `/bonita/7.12/`